Code/GraphMol/Chirality.cpp

```diff
@@ -1883,7 +1882,19 @@ void assignStereochemistry(ROMol &mol, bool cleanIt, bool force,
   for (ROMol::BondIterator bondIt = mol.beginBonds(); bondIt != mol.endBonds();
        ++bondIt) {
     if (cleanIt) {
-      if ((*bondIt)->getBondType() == Bond::DOUBLE) {
+      // enforce no stereo on small rings
+      if (!shouldDetectDoubleBondStereo(*bondIt) &&
+          ((*bondIt)->getBondType() == Bond::DOUBLE ||
+           (*bondIt)->getBondType() == Bond::AROMATIC)) {
```

Comment:
I'd reverse the order of this test since the `getBondType()` tests are simpler and cheaper:
```suggestion
      if (((*bondIt)->getBondType() == Bond::DOUBLE ||
           (*bondIt)->getBondType() == Bond::AROMATIC) && 
           !shouldDetectDoubleBondStereo(*bondIt)) {
```